src/test/java/com/ibanity/apis/client/helpers/KeyToolHelper.java

```diff
@@ -0,0 +1,69 @@
+package com.ibanity.apis.client.helpers;
+
```

Comment:
why in src/test? Use in source as a utility method?

Reply:
Because it doesn't cover all the possible ways of loading private keys/certificates and it adds dependency to external libs (like bountycastle). 
As we decided to remove the load of the certificates/keys from the lib, we don't need it in the production code but I kept it in the test code so customer can still be inspired.